configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
@jart Also see [here](https://github.com/tensorflow/tensorflow/issues/4807#issuecomment-255889425) and [here](https://github.com/tensorflow/tensorflow/issues/4807#issuecomment-256783132) regarding the previously stated preference by a Googler that this be made optional.